package.json

```diff
@@ -64,7 +64,6 @@
   "devDependencies": {
     "@cypress/code-coverage": "^3.9.7",
     "@istanbuljs/nyc-config-typescript": "^1.0.1",
-    "@mermaid-js/mermaid-cli": "^8.8.4",
```

Comment:
Was mermaid causing problems? I believe it is needed to run `npm run doc:diagrams`

Reply:
Hm, I think I had some problems installing it locally, but when I added it back, it worked. Maybe some minor version upgrade helped.